docs/semgrep-assistant/customize.md

```diff
@@ -0,0 +1,205 @@
+---
+slug: customize
+title: Customize Assistant
+hide_title: true
+description: Learn how to enable and configure Assistant's features.
+tags:
+  - Deployment
+  - Semgrep Assistant
+---
+
+# Customize Semgrep Assistant
+
+You can customize Semgrep Assistant by enabling and using the features detailed on this page.
+
+## Remediation
+
+Assistant remediation allows you to receive AI-generated code snippets for true positives. Perform the following to enable it:
+
+1. Sign in to Semgrep AppSec Platform, and navigate to **Settings > Deployment**.
+2. In the **Assistant** section, click the **Remediation** <i class="fa-solid fa-toggle-large-on"></i> if it is not yet enabled.
+3. *Optional*: Select a **confidence level** in the drop-down box. This value determines the level of quality at which the autofix code appears as a suggestion. A lower confidence level means that Semgrep Assistant displays the autofix suggestion even when the code quality may be incorrect.
+
+:::tip
+Semgrep recommends setting a low confidence level since even incorrect suggestions may be useful starting points for triage and remediation.
+:::
+
+## Auto-triage
+
+If [auto-triage](/semgrep-assistant/overview/#auto-triage), which allows you to get notifications whenever Assistant indicates a finding may be safe to ignore, isn't enabled for your deployment, you can do so as follows:
+
+1. Sign in to Semgrep AppSec Platform, and navigate to **Settings > Deployment**.
+2. In the **Assistant** section, click the **Auto-triage for Code** <i class="fa-solid fa-toggle-large-on"></i> if it is not yet enabled.
+3. Select whether you want alerts included in your **PR/MR comments** and **Slack notifications**.
+
+![MR comment from Semgrep Assistant in GitLab](/img/assistant-gl-comment.png#md-width)
+*Figure*. MR comment from Semgrep Assistant in GitLab.
+
+### Missing PR and comments
+Semgrep Assistant messages only appear in your PR comments for rules that are set to Comment or Block mode on the Rule Management page. Ensure that:
+
+* You have set rules to Comment or Block mode.
+  ![ Policies modes](/img/semgrep-assistant-comment.png#md-width)
+* You have selected PR/MR comments in **Semgrep AppSec Platform > Settings > Deployment** in the **Code** section.
+
+## Weekly priority emails
+
+If [weekly priority emails](/semgrep-assistant/overview/#weekly-priority-emails), which allows organization admins to receive information on top backlog tasks according to Assistant, isn't enabled for your deployment, you can do so as follows:
```

Comment:
Similar to the comment above (is this feature enabled by default in a given deployment?) - It seems weird to start an instruction with "if".